benchmarks/models.py

```diff
@@ -261,6 +262,10 @@ class ResultData(models.Model):
     result = models.ForeignKey(Result, related_name="data")
     benchmark = models.ForeignKey(Benchmark, related_name="data")
 
+    # can't be a proper ForeignKey because it breaks django_dynamic_fixtures
+    # (maybe because the primary key is not autogenerated?)
+    test_job_id = models.CharField(max_length=100, blank=False, null=True)
```

Comment:
it is, but when I added a proper ForeignKey attribute DDF would break horribly. I will try again, maybe I did something else wrong at the time.